app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -698,6 +699,13 @@ class Product extends \Magento\ImportExport\Model\Import\Entity\AbstractEntity
      */
     private $catalogConfig;
 
+    /**
+     * Source Item Importer
+     *
+     * @var
```

Comment:
please specify data type for `@var` StockItemImporterInterface

Reply:
Good spot - [fixed here](https://github.com/magento-engcom/msi/pull/211/commits/290d9193ac1a21423b8a36fad8c3c8115d0ff502)